src/components/ReviewsComponent/SingleReviewComponent.jsx

```diff
@@ -25,17 +20,17 @@ const SingleReviewComponent = ({
       starsArray.push(starEmpty);
     }
 
-    return starsArray.map((element) => element);
+    return starsArray.map((element, index) => (
+      <span key={index}>{element}</span>
```

Comment:
Element.id would better as a key.

Reply:
It couldn't be because the element corresponds to a star emoji and there must be 5 of them, therefore all 5 span would have the same key

Reply:
Ok, np, if there is time, we could work on enhancements.